examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/PPORun.py

```diff
@@ -0,0 +1,154 @@
+import wandb
+
+from stable_baselines3 import PPO
+from stable_baselines3.common.env_checker import check_env
+from stable_baselines3.common.callbacks import CheckpointCallback
+from typing import Callable
+
+import ray
+from ray import tune
+from ray.tune.registry import register_env
+import ray.rllib.agents.ppo as ppo
+from ray.tune.logger import DEFAULT_LOGGERS
+from ray.tune.integration.wandb import WandbLoggerCallback
+
+from khr3hvEnv import khr3hvRobotSupervisor
+
+
+# Scheduler for the learning rate when using the stable baselines
+def linear_schedule(initial_value: float) -> Callable[[float], float]:
+    """
+    Linear learning rate schedule.
+    :param initial_value:
+    :return: current learning rate depending on remaining progress
+    """
+    if isinstance(initial_value, str):
+        initial_value = float(initial_value)
+
+    # TODO  decrease the lr by .10
+    def func(progress_remaining: float) -> float:
+        """
+        Progress will decrease from 1 (beginning) to 0
+        :param progress_remaining: (float)
+        :return: (float)
+        """
```

Comment:
Improved the docsring.

```suggestion
        """
        Progress will decrease from 1 (beginning) to 0
        :param progress_remaining: The remaining progress.
        :type progress_remaining: float
        :return: Current learning rate depending on remaining progress.
        :rtype: float
        """
```